smoke-testNuGet.Config

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
-    <clear />
     <add key="source-built-packages" value="SOURCE_BUILT_PACKAGES" />
```

Comment:
Is this intentional? Should keep the clear if possible.

Reply:
Unintentional.  Thanks for catching it.